client/blocks/image-editor/test/image-editor-canvas.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+class ImageEditorCropMock extends Component {
+	constructor( props ) {
+		super( props );
+	}
+	render() {
+		return <dfn />;
+	}
+}
+
+describe( 'ImageEditorToolbar', () => {
+	useFakeDom();
+
+	let ImageEditorCanvas,
+		isImageEditorImageLoaded,
+		wrapper;
+
+	useSandbox( ( sandbox ) => {
+		isImageEditorImageLoaded = sandbox.stub().returns( true );
+	} );
+
+	useMockery( mockery => {
+		mockery.registerMock( 'state/ui/editor/image-editor/selectors', {
+			isImageEditorImageLoaded
+		} );
+		mockery.registerMock( './image-editor-crop', ImageEditorCropMock );
+	} );
+
+	before( () => {
+		ImageEditorCanvas = require( '../image-editor-canvas' ).ImageEditorCanvas;
+	} );
+
+	beforeEach( () => {
+		wrapper = shallow( <ImageEditorCanvas isImageLoaded={ true } /> );
+	} );
+
+	it( 'should render <ImageEditorCrop /> when this.props.isGreaterThanMinimumDimensions === `true`', () => {
```

Comment:
Let's try and be careful with our describes. Tests can often be documentation for 🔍 what the original intent of a piece of code was, so it's important to capture that over exactly what the code does. For example we might phrase this as, `should render crop controls when image is large enough to edit`, and `should not render crop controls when image is too small to edit`